code/index.html

```diff
@@ -1,22 +1,96 @@
+    <div class="text-container">
+      <h2 class="big-news-title">BIG NEWS</h2>
+      <p class="big-news-text">
+        Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.Lorem ipsum is a pseudo-Latin text used in web design, typography,
+        layout, and printing in place of English to
+        emphasise design elements over content.
+      </p>
+    </div>
+  </section>
+
+  <hr>
+  <section class="flex-parent">
+    <div class="flex-child blue">
+      <img src="./paper8.jpg" alt="Germany">
+      <p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.
+      </p>
+      <button>read more</button>
+    </div>
+    <div class="flex-child green">
+      <img src="./paper1.jpeg" alt="croatia">
+      <p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.</p>
+      <button>Read more</button>
+    </div>
+    <div class="flex-child yellow">
+      <img src="./paper2.jpeg" alt="sea">
+      <p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.</p>
+      <button>read more</button>
+    </div>
+    <div class="flex-child yellow">
+      <img src="./paper4.jpg" alt="seaburn">
+      <p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.</p>
+      <button>read more</button>
+    </div>
+    <div class="flex-child yellow">
+      <img src="./paper.jpeg" alt="reading newspaper">
+      <p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.</p>
+      <button>read more</button>
+    </div>
+    <div class="flex-child yellow">
+      <img src="./paper5.jpeg" alt="sas">
+      <p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.</p>
+      <button>read more</button>
+    </div>
+    <div class="flex-child yellow">
+      <img src="./paper6.jpeg" alt="sas">
+      <p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.</p>
+      <button>read more</button>
+    </div>
+    <div class="flex-child yellow">
+      <img src="./paper7.jpeg" alt="sas">
+      <p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.</p>
+      <button>read more</button>
+    </div>
+
+  </section>
+</body>
+
+</html>
```

Comment:
hmtl is really easy to read! Good structure with the indents and so. YOU ARE A PRO! :-)